pkg/cidata/cidata.TEMPLATE.d/boot/60-propagate-timezone.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -eux
+
+tz=$LIMA_TIMEZONE
+
+if [ -n "$tz" ]; then
+    test -f /usr/share/zoneinfo/$tz || exit 0 # incase tzdate is not installed, shouldn't block startup
```

Comment:
I don't like the `|| exit 0` part hidden in the middle of the line. It should either be a proper `if` block, or just fold it into the condition of the outer block:

```suggestion
if [ -n "$tz" ] && [ -f "/usr/share/zoneinfo/$tz" ]; then
```

Reply:
Took suggestion in [97c113c](https://github.com/lima-vm/lima/pull/1931/commits/97c113cde70a6db042accad70143046000a44da2).